cli/wallet/nep17.go

```diff
@@ -413,7 +413,12 @@ func transferNEP17(ctx *cli.Context) error {
 	defer wall.Close()
 
 	fromFlag := ctx.Generic("from").(*flags.Address)
-	from := fromFlag.Uint160()
+	var from util.Uint160
+	if fromFlag.IsSet {
+		from = fromFlag.Uint160()
+	} else {
+		from = wall.GetChangeAddress()
```

Comment:
It can return zero. Most probably `getDecryptedAccount` will fail then, but still it's be nice to separate this error from `can't find account for the address`.

Reply:
Fixed.